package.json

```diff
@@ -85,7 +85,6 @@
     "eth-block-tracker": "^4.1.0",
     "eth-contract-metadata": "^1.9.2",
     "eth-ens-namehash": "^2.0.8",
-    "eth-hd-keyring": "github:brave/eth-hd-keyring",
```

Comment:
not sure why this is here in the first place since eth-hd-keyring is only used as a dependency of eth-keyring-controller AFAICT